subprojects/launcher/src/main/java/org/gradle/launcher/daemon/server/health/DaemonMemoryStatus.java

```diff
@@ -36,10 +36,10 @@
     private final int nonHeapUsageThreshold;
     private final double thrashingThreshold;
 
-    public DaemonMemoryStatus(DaemonHealthStats stats, int heapUsageThreshold, double heapRateThreshold, int nonHeapUsageThreshold, double thrashingThreshold) {
+    public DaemonMemoryStatus(DaemonHealthStats stats, int heapUsageThreshold, double gcRateThreshold, int nonHeapUsageThreshold, double thrashingThreshold) {
         this.stats = stats;
         this.heapUsageThreshold = heapUsageThreshold;
-        this.heapRateThreshold = heapRateThreshold;
+        this.heapRateThreshold = gcRateThreshold;
```

Comment:
```suggestion
        this. gcRateThreshold = gcRateThreshold;
```

Did you want to rename the field as well?

Reply:
Yes, I did :)